shell/linters/prettier.sh

```diff
@@ -7,16 +7,16 @@ source "$DIR/languages/nodejs.sh"
 
 # Why: Used by the script that calls us
 # shellcheck disable=SC2034
-extensions=(yaml yml json md)
+extensions=(yaml yml json md ts)
 
 prettier_linter() {
   yarn_install_if_needed >/dev/null
-  git ls-files '*.yaml' '*.yml' '*.json' '*.md' | xargs -n40 "node_modules/.bin/prettier" -l --loglevel warn
+  git ls-files '*.yaml' '*.yml' '*.json' '*.md' '*.ts' | xargs -n40 "node_modules/.bin/prettier" -l --loglevel log || exit $?
 }
 
 prettier_formatter() {
   yarn_install_if_needed >/dev/null
-  git ls-files '*.yaml' '*.yml' '*.json' '*.md' | xargs -n40 "node_modules/.bin/prettier" --write --loglevel warn
+  git ls-files '*.yaml' '*.yml' '*.json' '*.md' '*.ts' | xargs -n40 "node_modules/.bin/prettier" --write --loglevel warn
```

Comment:
we used to format .ts files only for [`/api/clients/node`](https://github.com/getoutreach/devbase/pull/316/files#diff-22f948702707e5ab2dff115b8860ed6a0b368098b5e2a4dec22ed76be954e5e7L90-L103). Is there any concern to check lint and format .ts for entire repo?

Reply:
Should be fine!